datastore/store_test.go

```diff
@@ -410,3 +410,50 @@ func TestBlobStore_GetActiveSet(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, codec.MustEncode(as), got)
 }
+
+func TestCachedDb_GetEpochWeight(t *testing.T) {
```

Comment:
it makes sense to drop this method, i found that it is used only in:
- handler_test.go
- tortoise/core.go

in both cases it accidentally is using CacheDb

Reply:
Removed